Apps/phradar/radar.json

```diff
@@ -10,12 +10,13 @@
     "product_name": "Radar",
     "product_version_regex": ".*",
     "publisher": "RADAR, LLC",
-    "license": "Copyright (c) 2020 Splunk Inc.",
+    "license": "Copyright (c) 2020-2021 Splunk Inc.",
     "app_version": "1.0.1",
     "python_version": "3",
     "package_name": "phantom_radar",
     "main_module": "radar_connector.py",
-    "min_phantom_version": "4.8.24304",
+    "min_phantom_version": "4.9.39220",
```

Comment:
Why bump the phantom version here? Shouldn't it be assumed that our team completed QA with version 4.8 and changes with 4.9 would be untested?

Reply:
According to the most recent Splunk> Phantom update, the minimum Phantom version should be 4.9. As a result, we changed the pipeline and added checks to verify it. You can see that the pipeline is failing in your first commit for the same reason. 
Hence, contributors can check pipeline faults and apply fixes accordingly.